src/instrumentations/amqplib/AmqplibInstrumentation.ts

```diff
@@ -0,0 +1,24 @@
+import { Span } from '@opentelemetry/api';
+import {
+  AmqplibInstrumentation,
+  ConsumeInfo,
+  PublishInfo,
+} from '@opentelemetry/instrumentation-amqplib';
+import { Instrumentor } from '../instrumentor';
+
+export default class LumigoAmqplibInstrumentation extends Instrumentor<AmqplibInstrumentation> {
+  getInstrumentedModule(): string {
+    return 'amqplib';
+  }
+
+  getInstrumentation(): AmqplibInstrumentation {
+    return new AmqplibInstrumentation({
+      publishHook: (span: Span, publishInfo: PublishInfo) => {
+        span.setAttribute('messaging.publish.body', publishInfo.content.toString());
```

Comment:
can you limit it?
use the util `add_body_attribute`

Reply:
@saartochner did you mean use something like the `concatenatePayload` method you implemented for grpc? should that not be a general utility method for all the instrumentations?

Reply:
nope, check out the function `add_body_attribute` - I put it into utils 👍

Reply:
that's the python distro :)